Tools/eslint-config-cesium/README.md

```diff
@@ -0,0 +1 @@
+A [shareable ESLint config](http://eslint.org/docs/developer-guide/shareable-configs) for [Cesium](https://cesiumjs.org/).
```

Comment:
Change this to something like `The official [shareable ESLint config](http://eslint.org/docs/developer-guide/shareable-configs) for the [Cesium](https://cesiumjs.org/) ecosystem.  Also include a brief description of each usable config.  See https://www.npmjs.com/package/eslint-config-airbnb for inspiration (but ours should be much more succinct).